pkg/webservice/handlers.go

```diff
@@ -60,6 +60,27 @@ const (
 	NodeDoesNotExists        = "Node not found"
 )
 
+var allowedActiveStatusMsg string
+var allowedAppActiveStates map[string]bool
+
+func init() {
+	allowedAppActiveStates = make(map[string]bool)
+
+	allowedAppActiveStates["new"] = true
+	allowedAppActiveStates["accepted"] = true
+	allowedAppActiveStates["starting"] = true
+	allowedAppActiveStates["running"] = true
+	allowedAppActiveStates["completing"] = true
+	allowedAppActiveStates["failing"] = true
+	allowedAppActiveStates["resuming"] = true
+
+	var activeStates []string
+	for k := range allowedAppActiveStates {
+		activeStates = append(activeStates, k)
+	}
+	allowedActiveStatusMsg = fmt.Sprintf("Only following active states are allowed: %s", strings.Join(activeStates, ","))
```

Comment:
Could you change `active states` to `active statuses`? The query key is `status` so I prefer naming consistency.

Reply:
@chia7712 Thanks for review. In order to keep consistency, should change allowedAppActiveStates to allowedAppActiveStatuses?

Reply:
yep!